lib/pbench/server/api/resources/datasets_metadata.py

```diff
@@ -159,8 +175,9 @@ def _put(self, params: ApiParams, _) -> Response:
                 Metadata.setvalue(key=k, value=v, dataset=dataset, user=user)
             except MetadataError as e:
                 self.logger.warning("Unable to update key {} = {!r}: {}", k, v, str(e))
-                failures.append(k)
-        if failures:
+                fail_count += 1
+
+        if fail_count:
```

Comment:
A counter is better than a list, but why not a simple boolean?

Reply:
Because it goes to 11?